composer.json

```diff
@@ -24,7 +24,9 @@
     },
     "require-dev": {
         "league/oauth2-facebook": "^1.1",
-        "symfony/security-guard": "^2.8|^3.0"
+        "symfony/security-guard": "^2.8|^3.0",
+        "phpunit/phpunit": "^4.8",
```

Comment:
My question is why do we really need `phpunit/php-code-coverage` in this project? I agree with PHPUnit, we use it for tests, but we don't use `php-code-coverage` here. As for me adding a dependency which we don't use in a project just in case is a bad idea which leads to overhead and even could confuse other devs.